doc/dev/cephfs-mirroring.rst

```diff
@@ -210,8 +210,44 @@ Mirror Daemon Status
 --------------------
 
 Mirror daemons get asynchronously notified about changes in file system mirroring status
-and/or peer updates. CephFS mirror daemons provide admin socket commands for querying
-mirror status. To check available commands for mirror status use::
+and/or peer updates.
+
+CephFS mirroring module provides `mirror daemon status` interface to check mirror daemon
+status::
+
+  $ ceph fs snapshot mirror daemon status <fs>
```

Comment:
The doc is not consistent about the naming, sometimes the filesystem name is referred to as `<fs>` and sometimes `<fs_name>`, would be nice to fix this at some point.

Reply:
oh yeh! I was thinking the same when writing this line

;P